src/encord_active/app/data_quality/sub_pages/summary.py

```diff
@@ -1,19 +1,12 @@
 from typing import List
 
-import streamlit as st
-
 from encord_active.app.common.components.metric_summary import (
     render_data_quality_dashboard,
     render_label_quality_dashboard,
-    render_metric_summary,
```

Comment:
Perhaps we should remove the unused function then?

Reply:
Makes sense!